tests/test_all_tasks.py

```diff
@@ -3,21 +3,49 @@
 
 import pytest
 
-import catwalk.tasks
 import catwalk.models
+import catwalk.tasks
 from catwalk.task import InstanceFormat
 from catwalk.tasks.huggingface import HFMCInstance
 
+from .util import suite_B, suite_C
+
+# These are tasks are known to fail for now due to an unreachable server.
+known_failures = {
+    "lambada_mt_en",
+    "lambada_mt_fr",
+    "lambada_mt_de",
+    "lambada_mt_it",
+    "lambada_mt_es",
+    "triviaqa",
+}
```

Comment:
I had to `xfail` these tests because the server they rely on is apparently down. @dirkgr you might want to look into this more, maybe there's a new URL we should be using.

Reply:
Hmm. Once they are cached, we are much more robust to such failures.